apps/finance/buidler.config.js

```diff
@@ -0,0 +1,39 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+const hooks = require('./scripts/buidler-hooks')
+
+usePlugin('@aragon/buidler-aragon')
+usePlugin('@nomiclabs/buidler-solhint')
+usePlugin('buidler-gas-reporter')
+usePlugin('solidity-coverage')
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545',
+      accounts: {
+        mnemonic: "explain tackle mirror kit van hammer degree position ginger unfair soup bonus"
```

Comment:
We use the same mnemonic as in ganache-cli script

Reply:
Ah interesting, I made a note in one of the other apps, but maybe this is something we can export then.

This is just so the tests and the local development environment are using the same accounts, right? Technically this shouldn't have an impact? Or is the ganache-cli script from `contract-helpers` starting aragen?

Reply:
No, `contract-helpers` was not starting aragen. I was having this behavior: https://github.com/trufflesuite/truffle/issues/1062

Actually doing further tests it only happens if `ganache-cli` doesn't initialize a mnemonic. So the account configuration on `buidler.config` is not necessary. Regardless I wonder if it's a restriction on the way that buidler setup the JSON-RPC networks, I'll ask them 🤔